lib/facter/resolvers/networking_linux.rb

```diff
@@ -252,6 +253,11 @@ def retrieve_primary_interface
           @fact_list[:primary_interface] = primary_interface
         end
 
+        def retrieve_primary6_interface
+          primary_helper = Facter::Util::Resolvers::Networking::PrimaryInterface
+          primary6_interface = primary_helper.read_from_ip6_route
```

Comment:
This line seems to be redundant. I think that `@fact_list[:primary6_interface] = primary_helper.read_from_ip6_route` is sufficient.

Reply:
Yeah, I just wanted to follow the pattern used in `retrieve_primary_interface`.